modules/network/vpc/README.md

```diff
@@ -108,8 +108,6 @@ compact set of subnetworks possible.
 ```yaml
 - id: network1
   source: modules/network/vpc
-  settings:
-  - deployment_name: $(vars.deployment_name)
 ```
 
 This creates a new VPC network named based on the `deployment_name` variable
```

Comment:
There's a **_NOTE:_** that should be updated or removed as well.
I agree that it shouldn't be in the default example anymore, but I think it was initially included to demonstrate how to set some settings. 
It might be worth including `network_name: custom-vpc`(or something similar) as a secondary example; suggestion only, feel free to disregard.

Reply:
Removed the note and added alt setting to keep user aware of settings in general as per our offline discussion.